src/themes/shared/index.ts

```diff
@@ -28,6 +29,7 @@ export const shared = {
   fontWeights,
   fontSizes,
   fonts,
+  zIndices,
```

Comment:
While adding a new token values, you will also need to add the story for it as we have for the rest of the theme tokens.

![Screenshot 2022-11-08 at 12 38 37](https://user-images.githubusercontent.com/37380787/200554683-a89540c9-2999-4011-81b1-d82c1671e30f.png)

Thank you 🌸

Reply:
@GutierrezGo thanks, updated